api/hpa.go

```diff
@@ -39,7 +45,18 @@ func PutHpaResource(c *gin.Context) {
 		})
 		return
 	}
-	log.Info("Parse deployment succeeded")
+
+	err = scalingRequest.Validate()
+	if err != nil {
+		err := errors.Wrap(err, "Error parsing request:")
+		log.Error(err.Error())
```

Comment:
I know it's not how usually done in pipeline, but this is not actually an error from the application point of view, since it's only a validation failure of the data provided by the user. As such, it should logged as info or debug information, normally validation failure doesn't concern us (imagine an attack where the attacker deliberately sends invalid payloads just to flood us with error messages.)